test/fixtures.js

```diff
@@ -195,3 +195,47 @@ export const createColorList = (overrides = []) => [
   },
   ...overrides,
 ]
+
+export const createAnalysisConfig = (overrides = {}) => ({
+  label: '',
+  analysis: '',
+  color: [],
+  range: [],
+  variable: '',
+  category: '',
+  ...overrides,
+})
+
+export const createMatrixData = (overrides = {}) => ({
+  analysis: 'size_of',
+  category: 'measurements',
+  color: ['red', 'blue', 'white'],
+  data: [
+    {
+      day: 1,
+      sizeVariable: 30,
+    },
+    {
+      day: 45,
+      sizeVariable: 2,
+    },
+  ],
+  label: 'Size',
+  range: [1, 2],
+  stat: [
+    {
+      max: 30,
+      mean: 16,
+      min: 2,
+    },
+  ],
+  variable: 'sizeVariable',
```

Comment:
we should default these to fixtures to empty values too. That way there won't be confusion when reading the tests about where the data is coming from since it needs to be set when we instantiate the fixtures.

```suggestion
  analysis: '',
  category: '',
  color: [],
  data: [],
  label: '',
  range: [],
  stat: [],
  variable: '',
```

Reply:
[fba1285](https://github.com/AMP-SCZ/dpdash/pull/344/commits/fba128510aee9607c64d5855d8d8fcad4e94155f)